shared/locales/fr.js

```diff
@@ -46,9 +46,9 @@ module.exports = {
 
         charts: {
             Amount: 'Montant',
-            balance: 'balance',
-            by_category: 'par catégorie',
-            differences_all: 'rentrées et sorties (tous les comptes)',
+            balance: 'Solde',
+            by_category: 'Par catégorie',
+            differences_all: 'Rentrées et sorties (tous les comptes)',
```

Comment:
I prefer it capitalized, but it's a matter of tastes :p I can keep it without capitalization. Ate you OK with the 'relevé' translation (singulier)